convex/imageHttp.ts

```
@@ -0,0 +1,84 @@
+  getUrlParams,
+} from './httpUtils';
+import { Id } from './_generated/dataModel';
+
+/**
+ * HTTP action to upload images to Convex storage
+ */
+export const uploadImageHttp = httpAction(async (ctx, request) => {
+  // Validate authentication with user verification
+  const authResult = await validateAuthWithUser(request);
+  if (!authResult.success) {
+    return createErrorResponse(authResult.error!, 401);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)